AdobeStockImage/Model/Asset.php

```diff
@@ -0,0 +1,421 @@
+<?php
+
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\AdobeStockImage\Model;
+
+use Magento\AdobeStockImage\Api\Data\AssetInterface;
+use Magento\AdobeStockImage\Model\ResourceModel\Asset as ResourceModel;
+use Magento\Framework\Model\AbstractModel;
+
+class Asset extends AbstractModel implements AssetInterface
+{
+    /**
+     * Construct
+     */
+    protected function _construct()
+    {
+        $this->_init(ResourceModel::class);
+    }
+    /**
+     * Get media type id
+     * @return int
+     */
+    public function getMediaTypeId(): int
+    {
+        return (int)$this->getData("media_type_id");
+    }
+
+    /**
+     * Set media type id
+     * @param int $mediaTypeId
+     */
+    public function setMediaTypeId(int $mediaTypeId)
+    {
+        $this->setData("media_type_id", $mediaTypeId);
```

Comment:
Let's use constants instead of hardcoded strings for data keys

Reply:
Hi @sivaschenko, I added this change. Can you check again please?